ProjectManagmentSystem/src/Modules/AuthenticationModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,7 +1,147 @@
+
+    }
+
+  }
   return (
-    <div>ResetPass</div>
+    <div className='auth-container'>
+      <div className='container-fluid'>
+        <div className="row d-flex vh-100 justify-content-center align-items-center">
+          <div className="col-md-6">
+            <div className="forgot text-center mb-4">
+              <img src={logo} alt="logo" className='w-25' />
+            </div>
+
+            <form action="#" onSubmit={handleSubmit(onSubmit)} className='form-auth vh-25' style={{ padding: "30px 30px" }}>
+              <span className='welcome-pms'>welcome to PMS</span>
+              <h1 className='auth-title'>Reset  Password</h1>
+              <div className='auth-standard-basic'>
+                <span className='e-mail'>E-mail</span> <br />
+                <TextField className='mb-4 w-75' id="standard-basic" label="Enter your E-mail" variant="standard"
+                  type="text"
+                  {...register("email", {
+                    required: "Email is required",
+                    pattern: {
+                      value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+                      message: "Invlid mail"
+                    }
+                  })}
+                />
+              </div>
+              {errors.email && (
+                <p className="alert alert-danger">{errors.email.message} </p>
+              )}
+              <div className='auth-standard-basic'>
+                <span className='e-mail'>OTP Verification</span> <br />
+                <TextField className='mb-4 w-75' id="standard-basic" label="Enter Verification" variant="standard"
+                  type="text"
+                 {...register("seed", {
+                    required: "OTP is required",
+                    pattern: {
+                      message: "Invlid password"
+                    }
+                  })}
+                />
+              </div>
+              {errors.seed && (
+                <p className="alert alert-danger">{errors.seed.message} </p>
+              )}
+              <div className='auth-standard-basic'>
+                <span className='e-mail'>New Password</span> <br />
+                <TextField className='mb-4 w-75' id="standard-basic" label="Enter your New Password" variant="standard"
+                  type="password"
+                  {...register("password", {
+                    required: "password is required",
+                    pattern: {
+                      message: "Invlid password"
+                    }
+                  })}
+                />
+              </div>
+              {errors.password && (
+                <p className="alert alert-danger">{errors.password.message} </p>
+              )}
+              <div className='auth-standard-basic'>
+                <span className='e-mail'>confirm Password</span> <br />
+                <TextField className='mb-4 w-75' id="standard-basic" label="Confirm New Password" variant="standard"
```

Comment:
We need to make sure that the new password and confirm password match each other.